@ecomplus/storefront-components/src/html/TheProduct.html

```diff
@@ -389,6 +389,7 @@
               <shipping-calculator
                 :shippedItems="isKit ? kitItems : [{
                   ...body,
+                  ...shippingData || {},
```

Comment:
Mas não funcionaria só passando o `...this.selectedVariation` aqui? Em vez de criar esse `shippingData`...

Reply:
```
...body,
...selectedVariation,
product_id: body._id,
variation_id: selectedVariationId,
quantity: qntToBuy || body.min_quantity || 1
```

?

Reply:
Bom, eu pensei isso, mas o ID da variação iria sobrescrever o id do produto, por isso que eu tive que fazer essa volta toda para criar um shippingData, removendo o _id, usando lodash, porque pra deletar o id, falava que não tinha um setter, só que eu não poderia deletar o id originalmente da variação...
Seria problema modificar esse _id no calculo apenas, se não for, realmente não precisaria dar tanta volta.

Reply:
Nem usa o `_id` na verdade, usa `product_id` e esse não seria sobrescrito

Reply:
https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-components/src/js/ShippingCalculator.js#L25-L42